frontend/src/components/ViewTrips/save-trip-form-elements.js

```diff
@@ -112,9 +112,12 @@ function createMultiFormControl(refArr, placeholder, defaultEmailArr) {
  *                           input prop.
  * @param {string} formLabel Label/title for the form input.
  * @param {string} inputType Input type of the form.
- * @param {!React.RefObject} ref Ref attached to the values inputted in the form.
+ * @param {!React.RefObject|!Array<React.RefObject>} ref Ref attached to the
+ *     values inputted in the form. This is an array of `React.RefObject`s when
+ *     `inputType` is 'emails'.
  * @param {string} placeholder Placeholder text value in the form input.
- * @param {?string|?Array<string>} defaultVal Default value in the form input.
+ * @param {?string|?string[]} defaultVal Default value in the form input.
+ *     This is an array when `inputType` is 'emails'.
  * @return {JSX.Element} The Form.Group element.
```

Comment:
Same as previous comments about backticks.

Reply:
If you were talking about Form.Group in this case, yes that should have backticks. Fixed.